.buildkite/bk.integration.pipeline.yml

```diff
@@ -0,0 +1,163 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
+  ASDF_MAGE_VERSION: 1.14.0
+  ASDF_GOLANG_VERSION: 1.22.6
+  ASDF_TERRAFORM_VERSION: 1.9.3
+
+steps:
+  # - label: "Integration tests: packaging"
+  #   key: "package-it"
+  #   command: ".buildkite/scripts/steps/integration-package.sh"
+  #   artifact_paths:
+  #     - build/distributions/**
+  #   agents:
+  #     provider: "gcp"
+  #     machineType: "n1-standard-8"
+  - label: Start ESS stack for integration tests
+    key: integration-ess
+    # depends_on:
+    #   - package-it
+    command: |
+      #!/usr/bin/env bash
+      set -euo pipefail
+      source .buildkite/scripts/steps/ess_start.sh
+    artifact_paths:
+      - test_infra/ess/*.tfstate
+      - test_infra/ess/*.lock.hcl
+    agents:
```

Comment:
We could replace this with a container image, snip the `agent` section only

```
agents:
      image: "docker.elastic.co/ci-agent-images/platform-ingest/buildkite-agent-beats-ci-with-hooks:0.5"
      useCustomGlobalHooks: true
```

This is going to provide faster execution time for the step. Step time will go down from ~4m to less than 1m (excluding the time of the TF apply)

Reply:
Thanks! QQ: Is the version is intentionally set to `0.5` in the example?

Reply:
I don't recall the latest version; I think it's 0.5, but you can check the repo for the latest.